ironfish/src/webApi.ts

```diff
@@ -93,16 +93,16 @@ export class WebApi {
     return response.data
   }
 
-  async getNextFaucetTransaction(): Promise<FaucetTransaction | null> {
+  async getNextFaucetTransactions(count: number): Promise<FaucetTransaction[] | null> {
     this.requireToken()
 
     try {
-      const response = await axios.get<FaucetTransaction>(
-        `${this.host}/faucet_transactions/next`,
+      const response = await axios.get<{ object: string; data: FaucetTransaction[] }>(
+        `${this.host}/faucet_transactions/next?count=${count}`,
         this.options(),
       )
 
-      return response.data
+      return response.data.data
     } catch (e) {
       if (IsAxiosError(e) && e.response?.status === 404) {
```

Comment:
I think the API here is a bit strange. It seems to return null because the old API used to return null when it got a 404, which was used by the API to indicate there were no results left to process. In an API that sends a list, it should just send an empty list when there are no items to process.

Reply:
I've changed it to check for an empty list or a null value. This is to ensure that there is no downtime or errors when both the ability to return a single faucet request and the ability to return null are removed from the API. Once that happens, I can remove the null check and the empty list will be returned when there are no items to process.